File: java-api-core/src/com/kinvey/java/dto/UserNameExists.java
package com.kinvey.java.dto;

import com.google.api.client.util.Key;

/**
 * Created by yuliya on 06/01/17.
 */

public class UserNameExists {
    @Key("usernameExists")
    private boolean usernameExists;

    public boolean doesUsernameExist() {
        return usernameExists;
    }

    public void setUsernameExists(boolean usernameExists) {
        this.usernameExists = usernameExists;
    }
}

File: java-api-core/src/com/kinvey/java/dto/UserNameExists.kt
package com.kinvey.java.dto

import com.google.api.client.util.Key

/**
 * Created by yuliya on 06/01/17.
 */

class UserNameExists {
    @Key("usernameExists")
    private var usernameExists: Boolean = false

    fun doesUsernameExist(): Boolean {
        return usernameExists
    }

    fun setUsernameExists(usernameExists: Boolean) {
        this.usernameExists = usernameExists
    }
}
